packages/core/package.json

```diff
@@ -57,15 +57,15 @@
     "build:bundle": "rollup --config rollup.config.js",
     "fix": "run-s fix:*",
     "fix:prettier": "prettier . --write",
-    "fix:lint": "eslint src --ext .ts --ext .cjs --fix",
+    "fix:lint": "eslint src *.js --fix",
```

Comment:
maybe `eslint -ext .js,.ts --fix`
At first glance I thought we stopped checking for `.ts` 😓

Reply:
https://eslint.org/docs/latest/use/command-line-interface#--ext

> Default Value: .js and the files that match the overrides entries of your configuration.

Hence ts files are already covered so we don't have to dupe the value. If my understanding is correct...